### PR TITLE
feature(main): add status cmd for post check

### PR DESCRIPTION
**Pull request by cuisongliu**
Signed-off-by: cuisongliu <cuisongliu@qq.com>

### CODE REVIEW

**Comment by cuisongliu** [comment]
```
Registry Service Status
  Port: :5000
  DebugPort: :5001
  Storage: /var/lib/registry
  Delete: false
  RegistryDomain: sealos.hub:5000
  Htpasswd: admin:$2a$10$g4Dnl1gJ2i7FizmVVHRoEuklm37gW20qfYHIkhJkjw5dd5KrIifNu
  Auth: admin:passw0rd
  Ping: ok
  Error: <nil>
  Logger: journalctl -xeu registry

CRIShim Service Status
  Error: <nil>
  Logger: journalctl -xeu image-cri-shim
  Rootfs ImageList:
    - ghcr.io/labring/lvscare:v4.1.3
    - registry.k8s.io/coredns/coredns:v1.9.3
    - registry.k8s.io/etcd:3.5.4-0
    - registry.k8s.io/kube-apiserver:v1.25.0
    - registry.k8s.io/kube-controller-manager:v1.25.0
    - registry.k8s.io/kube-proxy:v1.25.0
    - registry.k8s.io/kube-scheduler:v1.25.0
    - registry.k8s.io/pause:3.8
  Config List:
    CRISocket: /run/containerd/containerd.sock
    RegistryAddress: http://sealos.hub:5000
    RegistryAuth: admin:passw0rd
    RootfsImageList: /var/lib/image-cri-shim
    ShimSocket: /var/run/image-cri-shim.sock

CRI Status
  Error: <nil>
  RegistryPullStatus: ok:Image is up to date for sha256:4e42fb3c9d90ed7895bc04a9d96fe3102a65b521f485cc5a4f3dd818afef9cef
  ImageShimPullStatus: ok:Image is up to date for sha256:4e42fb3c9d90ed7895bc04a9d96fe3102a65b521f485cc5a4f3dd818afef9cef
  CRICtl ImageList:
    - sealos.hub:5000/coredns/coredns:v1.9.3
    - sealos.hub:5000/etcd:3.5.4-0
    - sealos.hub:5000/kube-apiserver:v1.25.0
    - sealos.hub:5000/kube-controller-manager:v1.25.0
    - sealos.hub:5000/kube-proxy:v1.25.0
    - sealos.hub:5000/kube-scheduler:v1.25.0
    - sealos.hub:5000/pause:3.8
  ContainerList:
    ContainerID: 30d3dbb47679e  State: CONTAINER_RUNNING  Name: kube-proxy  Attempt: 0 PodName: kube-proxy-fsb5b
    ContainerID: f327ad071e606  State: CONTAINER_RUNNING  Name: kube-apiserver  Attempt: 0 PodName: kube-apiserver-default-node-1
    ContainerID: d93e23d9e4f4d  State: CONTAINER_RUNNING  Name: etcd  Attempt: 0 PodName: etcd-default-node-1
    ContainerID: 05856ce338164  State: CONTAINER_RUNNING  Name: kube-controller-manager  Attempt: 1 PodName: kube-controller-manager-default-node-1
    ContainerID: 9033e7a922bc1  State: CONTAINER_RUNNING  Name: kube-scheduler  Attempt: 1 PodName: kube-scheduler-default-node-1
  Logger: crictl logs -f CONTAINER-ID
  Config List:
    CRISocket: unix:///run/containerd/containerd.sock
    ShimSocket: unix:///var/run/image-cri-shim.sock

Systemd Service Status
  Logger: journalctl -xeu SERVICE-NAME
  Error: <nil>
  Init System List:
    Name: kubelet Status: Enable && Active
    Name: containerd Status: Enable && Active
    Name: cri-docker Status: Disable && NotActive
    Name: docker Status: Disable && NotActive
    Name: registry Status: Enable && Active
    Name: image-cri-shim Status: Enable && Active

Cluster Node Status
  ReadyNode: 0/1
  Not Ready Node List:
    NodeIP: 192.168.64.59

Cluster Pod Status
  Namespace: default
  RunningPod: 0/0
  Namespace: kube-node-lease
  RunningPod: 0/0
  Namespace: kube-public
  RunningPod: 0/0
  Namespace: kube-system
  RunningPod: 5/7
  Not Running Pod List:
    PodName: coredns-565d847f94-rwdhx
    PodName: coredns-565d847f94-z6xnm

Cluster Service Status
  Namespace: default
  HealthService: 1/1
  UnhealthServiceList:
  Namespace: kube-node-lease
  HealthService: 0/0
  UnhealthServiceList:
  Namespace: kube-public
  HealthService: 0/0
  UnhealthServiceList:
  Namespace: kube-system
  HealthService: 0/1
  UnhealthServiceList:
    ServiceName: kube-dns

Cluster Status
  Node List:
	NodeName: default-node-1
		NodeIP: 192.168.64.59
		KubeAPIServer: Running
		KubeControllerManager: Running
		KubeScheduler: Running
		KubeletErr: <nil>
```

File: pkg/checker/cluster_checker.go [hunk 1]
```diff
@@ -109,13 +109,14 @@ func (n *ClusterChecker) Output(clusterStatus []ClusterStatus) error {
 		KubeletErr: {{.KubeletErr}}
     {{- end }}
 `)
-	if err != nil {
-		panic(err)
+	if err != nil || !isOk {
+		if err != nil {
+			logger.Error("cluster checkers template can not excute %s", err.Error())
```
Comment (fanux) [inline pkg/checker/cluster_checker.go:114]: failed to render cluster checkers template.

File: cmd/sealos/cmd/status.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cluster, err := clusterfile.GetClusterFromName(clusterName)
+			if err != nil {
+				return fmt.Errorf("get default cluster failed, %v", err)
+			}
+			list := []checker.Interface{checker.NewRegistryChecker(), checker.NewCRIShimChecker(), checker.NewCRICtlChecker(), checker.NewInitSystemChecker(), checker.NewNodeChecker(), checker.NewPodChecker(), checker.NewSvcChecker(), checker.NewClusterChecker()}
+			return checker.RunCheckList(list, cluster, checker.PhasePost)
+		},
+	}
+	checkCmd.Flags().StringVarP(&clusterName, "cluster", "c", "default", "name of cluster to applied exec action")
```
Comment (fanux) [inline cmd/sealos/cmd/status.go:41]: to applied exec action？

Comment (cuisongliu) [reply cmd/sealos/cmd/status.go]: done

**Review by fanux — APPROVED** [review]

**Pull request merged**